lib/views/filter_widgets/timeline_filter/timeline_filters.dart

```diff
@@ -0,0 +1,13 @@
+/// This enum is used to create the timeline filters.
+/// It contains the name of the filters.
+enum TimeLineFilters {
+  nearest("Nearest"),
+  best("Best"),
+  newFilter("New"),
+  top("Top"),
+  hot("Hot");
```

Comment:
Filters are confusing. We should give them clearer names.

Reply:
I changed the filters to sort option, and also changed the `newFilter` instance to `lastest` since the `new` keyword is already reserved by Flutter.